src/TMGToolbox/inputOutput/importNetwork.py

```diff
@@ -143,6 +143,46 @@ def addDummyLink(transitVehicle, node, nextLink, transitLine, allVehicles):
     busStop.setPosition(linkLength/2) # stop at midpoint of link
     return newLink, busStop
 
+# Function to add curvature to a link
+def addLinkCurvature(link, pointsToAdd):
+    # insert points in reverse order after the origin
+    for point in reversed(pointsToAdd):
+        link.addPointAt(1,point)
```

Comment:
Why are we inserting into a list backwards?  Are we sure that there no addPoint variant that appends to the end, nor the ability call addPointAt at the end of the list?